README.md

```diff
@@ -126,6 +126,10 @@ beyond the LTS release.
 </tr>
 </table>
 
+## A note about v4.2 LTS
+
+While v4.2 was released in October, *this is a one-off event!* There was no LTS release plan in place in April 2015; v4.2 LTS was released in August in order to get v4.x updates into an LTS release as quickly as possible. In the future **LTS releases will be cut in April**.
```

Comment:
If this is too much for the main section/doc, perhaps it'd be better in a FAQ section or doc? Let me know if you'd like this in Q/A format (and whether in a separate .md file or not) and I'll put that together.